lib/dartino/device/device.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:async';
+
+import 'package:atom_dartlang/atom.dart';
+
+import '../dartino_util.dart';
+import '../launch_dartino.dart';
+import '../sdk/dartino_sdk.dart';
+import 'stm32f746disco.dart';
```

Comment:
I'm hoping that after a few boards, I can generalize, and then yes, the names will be more user friendly at that point :-)